jsk_image_capture/src/jsk_image_capture/jsk_image_capture.py

```diff
@@ -0,0 +1,69 @@
+# -*- encoding: utf-8 -*-
```

Comment:
change file name from `jsk_image_capture` to other (i.e. `jsk_image_capture_base`)
otherwise, python2 confuse importing `package` or `filename`.
this is the reason why you cannot import `jsk_image_capture.srv`

Reply:
this is actually quite important tip, but not common.
do not put a file which has the same name as the package name.